config.md

```diff
@@ -120,25 +120,26 @@ List of flags to remove from the compile command.
 In all cases, `-Xclang` is also removed where needed.
 
 Example:
+
 - Command: `clang++ --include-directory=/usr/include -DFOO=42 foo.cc`
 - Configuration: `Remove: [-I, -DFOO=*]`
 - Result: `clang++ foo.cc`
 
 Flags added by the same CompileFlags entry will not be removed.
 
-## Index
+### Index
 
 Controls how clangd understands code outside the current file.
 
-```
+```yaml
 Index:
   Background: Skip     # Disable slow background indexing of these files.
 ```
 
-clangd's indexes provide information about symbols that isn't available to
+clangd indexes provide information about symbols that isn't available to
```

Comment:
both are correct, but I prefer "clangd's" here as it's more specific about the relationship